build/ci/official.yml

```diff
@@ -61,117 +88,182 @@ steps:
     testRunTitle: 'Unit Test Results'
   condition: succeededOrFailed()
 
-- task: PowerShell@2
-  inputs:
-    filePath: build\variables\_vsts.ps1
-    failOnStderr: true
-  displayName: Set VSTS variables based on source 
-
-# Symbols is the one thing we cannot declare in the ps1 artifact scripts due to the extra steps
-# needed and the specific order it has to be performed in. Copy -> Publish Symbols -> Publish
-# Artifact -> publish to symweb
+###################################################################################################################################################################
+# PUBLISH SYMBOLS
+###################################################################################################################################################################
 
+# Symbols is the one thing we cannot declare in the ps1 artifact scripts due to the extra steps needed and the specific order it has to be performed in.
+# Copy -> Publish Symbols -> Publish Artifact -> Publish to SymWeb
 - task: CopyFiles@2
+  displayName: Copy Symbols
   inputs:
     SourceFolder: '$(Build.SourcesDirectory)\artifacts\$(BuildConfiguration)'
     Contents: |
       **/Microsoft.VisualStudio.ProjectSystem.Managed?(*.pdb|*.dll|*.xml)
       **/Microsoft.VisualStudio.AppDesigner?(*.pdb|*.dll|*.xml)
       **/Microsoft.VisualStudio.Editors?(*.pdb|*.dll|*.xml)
-    TargetFolder: $(Build.ArtifactStagingDirectory)/symbols
-  displayName: Copy symbols
+    TargetFolder: $(Build.ArtifactStagingDirectory)/Symbols
 
+# https://docs.microsoft.com/azure/devops/pipelines/tasks/build/index-sources-publish-symbols
 - task: PublishSymbols@2
-  displayName: Archive symbols to VSTS
+  displayName: Publish Symbols to Azure Artifacts
   inputs:
-    SymbolsFolder: $(Build.ArtifactStagingDirectory)/symbols
-    SearchPattern: 
-      '**/*.pdb
+    SymbolsFolder: '$(Build.ArtifactStagingDirectory)/Symbols'
+    SearchPattern: |
```

Comment:
TIL about the pipe in YAML: https://stackoverflow.com/a/18708156/24874

Reply:
This was an interesting find. The current search pattern was broken. If you look at any of our current builds, this task will say "0 files found". For example: https://dev.azure.com/devdiv/DevDiv/_build/results?buildId=5636220&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=8e355ed5-37f3-5c2a-bb8d-5efc727482d6&l=9

I don't know how long long it hasn't been working, but this change fixes in. In my test builds, you can see it actually publishing the symbols: https://dev.azure.com/devdiv/DevDiv/_build/results?buildId=5637304&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=8e355ed5-37f3-5c2a-bb8d-5efc727482d6&l=9

So, this is basically a bug fix in our pipeline.